src/Contracts/User.php

```diff
@@ -38,4 +38,11 @@ public function getEmail();
      * @return string
      */
     public function getAvatar();
+
+    /**
+     * Get the gender of the user.
+     *
+     * @return string
+     */
+    public function getGender();
```

Comment:
Is this targeting next major release? cause this potentially a huge breaking change.